### PR TITLE
In training ECAPA-TDNN，learning rate adjustment

### PR DESCRIPTION
**Pull request by qiny1012**
The learning rate is changed only on the training set and not on the verification set

### CODE REVIEW

**Comment by mravanelli** [comment]
Thank you @qiny1012 for the fix. I just fixed the linters and now all the tests should pass.
